docker-compose.yaml

```diff
@@ -141,16 +142,34 @@ services:
     depends_on:
       - keycloak
 
-  keycloak: # https://hub.docker.com/r/jboss/keycloak
+  postgres: # https://hub.docker.com/_/postgres
+    container_name: keycloak-postgres
+    image: postgres:${POSTGRES_VERSION:-14.5-alpine}
+    environment:
+      POSTGRES_DB: bitnami_keycloak
+      POSTGRES_USER: bn_keycloak
+      POSTGRES_PASSWORD: "#3$]O?4RGj)DE7Z!9SA5"
```

Comment:
This password is using `$` which is used for variable interpolation, so either escape it with another `$` or replace the `$` with another character.